### PR TITLE
Comment out exceptions for plugins without wiki pages.

### PR DESCRIPTION
**Pull request by abayer**
Also adds .idea to .gitignore, because, well, yeah.

cc @daniel-beck, @orrc, @batmat, @reviewbybees

### CODE REVIEW

**Comment by orrc** [comment]
Yay! :tada:

Though I would just delete everything after line 69.
Git history is there for reference, and on the mailing lists I believe we always linked to a commit, rather than master.

**Comment by abayer** [comment]
I'd like to leave the list there for a while, just to make it easier to remember what to link people to, but I'm not married to it one way or the other. And leaving an explanation *somewhere* in the repo is a good idea, IMO.

**Comment by batmat** [comment]
+1 merge it! :-)

+1 with @orrc too. IMO SCM is there for removal and commenting out should be avoided (it's the _pool man's scm_ ;-)).  

For reference purpose, it's always possible to reference the [commit itself](https://github.com/abayer/backend-update-center2/commit/0b01634e7e13cd2940c5f7209199cd0fb199b125) with the associated message.

Thanks Andrew

**Comment by abayer** [comment]
New push removing them rather than commenting. =)

**Comment by recena** [comment]
If my opinion counts... :+1:

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ target
 .settings
 www
 .htaccess
+.idea
```
Comment (batmat) [inline .gitignore]: (nit-picking: missing newline)

Comment (abayer) [inline .gitignore]: And fixed. =)

**Comment by batmat** [comment]
LGTM!

**Comment by abayer** [comment]
I'm gonna give it 'til tomorrow morning US time to merge, since I said originally that this would happen on the 29th. =)

**Comment by batmat** [comment]
J-1 : :birthday: :wink:

**Comment by abayer** [comment]
...yes, it's my birthday present to the Update Center. Cleaning!

**Comment by oleg-nenashev** [comment]
:+1: :bee:

**Comment by recena** [comment]
Exactly, :bee:

**Comment by andresrc** [comment]
:bee:

**Comment by abayer** [comment]
@reviewbybees done

**Pull request merged**